### PR TITLE
Meals Tables

### PR DESCRIPTION
**Pull request by emcooper**
Completes the following user stories:

- Viewing meal tables
- Foods in meal tables
- Total calories in meal tables
- Remaining calories in meal tables
- Remaining calories more than zero
- Remaining calories less than zero
- Totals table
- Styling Remaining Calories in Totals table

Later in the project I hope to refactor to incorporate an OO approach.

I wrote a unit test but cannot get Mocha to recognize the jQuery despite going through various stack overflow solutions. @case-eee any suggestions for that?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,25 @@
 <head>
   <meta charset="utf-8">
   <title>Quantified Self - Diary</title>
+  <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```
Comment (case-eee) [inline index.html:6]: i suggest using webpack to load this in for you!

File: index.html [hunk 2]
```diff
@@ -3,9 +3,25 @@
...
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
 </head>
 <body>
 
   <script src="main.bundle.js"></script>
+<div class="container">
```
Comment (case-eee) [inline index.html:14]: be mindful of your indentation here!

File: test/meal-test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const assert = require('chai').assert;
+const meals = require('../lib/meals.js')
+global.$ = require('jquery')(window);
```
Comment (case-eee) [inline test/meal-test.js]: i believe _if_ you're using webpack to load jquery, you can simply require it here... 

```js
const $ = require('jquery')
```

**Review by case-eee — COMMENTED** [review]
@emcooper see my comments and let me know if that helps answer your question 👍

File: test/meal-test.js (hunk 3 above)
Comment (emcooper) [reply test/meal-test.js]: Thanks! I will try that.

Comment (emcooper) [reply test/meal-test.js]: Webpack didn't seem to fix it. We had a JS questions workshop and apparently this has been an ongoing issue with no easy fix, so I'll put it on hold for now.

File: index.html (hunk 1 above)
Comment (jtruong2) [reply index.html:6]: I have webpack and my jquery doesn't work without this line of code

**Pull request merged**